cl/corpus_importer/tasks.py

```diff
@@ -632,11 +632,14 @@ def get_and_process_free_pdf(
         logger.info(f"{msg} Retrying.")
         raise self.retry(exc=exc)
 
+    pdf_bytes = None
+    if r:
+        pdf_bytes = force_bytes(r.content)
```

Comment:
What's this doing?

Reply:
Yeah, since now we can get a PDF document either directly from an HTTP Response or from a `ProcessingQueue` file I changed the response `update_rd_metadata` param from a `response: Optional[Response]` to `pdf_bytes: Optional[bytes]` and spread the changes upstream where `update_rd_metadata` is called, so this fragment of code just take the Response content, force to bytes before passing to `update_rd_metadata`.

I tried to use something like `response: Optional[Response | bytes]` but mypy was not happy with that. So I thought the best option is to update the method in all the places where is used.

Reply:
Why the force_bytes though? Isn't r.content always bytes anyway?

Reply:
Yeah, you're right in this case it'll be always bytes. I got confused by this comment: https://github.com/freelawproject/courtlistener/blob/main/cl/corpus_importer/tasks.py#L1761
But yes here there is no need to force it to bytes, I'll change it.

Reply:
I put that comment there ages ago, and yeah, it's confusing. It's not up to my standards now and I'd want to figure out why r.content wasn't always bytes. Maybe it was a py2 thing? I don't know, but that comment seems pretty wrong to me now.